charts/darts-portal/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: '1.0'
 description: A Helm chart for darts-portal App
 name: darts-portal
 home: https://github.com/hmcts/darts-portal
-version: 0.0.25
+version: 0.0.26
```

Comment:
Why are we changing this? I don't know what this is for

Reply:
We needed to change as I have updated the values files in charts.